analytics-listeners/src/main/java/org/exoplatform/analytics/listener/jcr/JCRNodeListener.java

```diff
@@ -80,6 +81,16 @@
 
   private static final Set<String>              CURRENTLY_PROCESSING_NODE_PATH_QUEUE = new HashSet<>();
 
+  private static final String                   EXO_USER_PREFERENCES                 = "exo:userPrefferences";
+
+  private static final String                   EXO_EDITORS_RUNTIME_ID               = "exo:editorsId";
+
+  private static final String                   EXO_CURRENT_PROVIDER                 = "exo:currentProvider";
+
+  private static final String                   EXO_PREFERRED_EDITOR                 = "exo:prefferedEditor";
+
+  private static final List<String>             excludedPropertyNames                = List.of(EXO_EDITORS_RUNTIME_ID, EXO_CURRENT_PROVIDER, EXO_PREFERRED_EDITOR);
+
```

Comment:
We may have other properties to ignore later, let's make this possible by configuration. We should not hardcode them in this Listener